### ISSUE TITLE
bug(auth) isLoaded incorrect behaviour

### ISSUE THREAD

**Issue by Modestas** [prescottprue/react-redux-firebase#290]
**Do you want to request a *feature* or report a *bug*?**  
Bug

**What is the current behavior?**
```
@firebaseConnect()
@connect(({ firebase: { auth, profile } }) => ({
  auth,
  user: profile
}))
```
Can't show spinner on auth loading because `isLoaded` state is behaving incorrectly.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [codesandbox](https://codesandbox.io/) or similar.**
Not loaded auth - `auth: { isLoaded: false, isEmpty: true}`
_[bug]_ Loaded but not logged in - `auth: { isLoaded: false, isEmpty: true}`
Loaded and logged in - `auth: { isLoaded: true, isEmpty: false}`

**What is the expected behavior?**
Not loaded auth - `auth: { isLoaded: false, isEmpty: true}`
Loaded but not logged in - `auth: { isLoaded: true, isEmpty: true}`
Loaded and logged in - `auth: { isLoaded: true, isEmpty: false}`

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
2.0.0-beta.8 and 2.0.0-beta.9

**Comment by prescottprue** [comment]
@Modestas This is definitely an issue thanks for reporting.

The change to separated reducers in v2.0.0 means that default state is required (which is why things are different than v1). It seems that in that change we did not correctly have auth state updating.

**Comment by prescottprue** [comment]
@Modestas The fix for this was included in v2.0.0-beta.10. Let me know if still does not function as you expect. Going to add more extensive testing for the `onAuthStateChange` (where it seemed that this issue came from) stuff before `v2.0.0` is out of pre-release.
